src/core/plugins/layout/spec-extensions/wrap-selector.js

```diff
@@ -9,12 +9,12 @@ export const taggedOperations = (oriSelector, system) => (state, ...args) => {
   // Filter, if requested
   let filter = layoutSelectors.currentFilter()
   if (filter) {
-    if (filter !== true && filter !== "true" && filter !== "false") {
+    if (filter !== true) {
       taggedOps = fn.opsFilter(taggedOps, filter)
     }
   }
   // Limit to [max] items, if specified
-  if (maxDisplayedTags && !isNaN(maxDisplayedTags) && maxDisplayedTags >= 0) {
+  if (typeof maxDisplayedTags === "number" && maxDisplayedTags >= 0) {
```

Comment:
When https://github.com/swagger-api/swagger-ui/pull/9829/files#r1567975985 is fixed, `typeof maxDisplayedTags === "number"` should not be needed here anymore